install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -451,6 +456,23 @@ spec:
         configMap:
           name: {{ .Values.global.cni.configMap }}
 {{- end }}
+{{- if and (.Values.global.hubble.enabled) (.Values.global.hubble.listenAddress) (.Values.global.hubble.tls.enabled) }}
+      - name: hubble-tls
+        projected:
+          sources:
+          - secret:
+              name: hubble-server-certs
+              items:
+                - key: tls.crt
+                  path: server.crt
+                - key: tls.key
+                  path: server.key
```

Comment:
I thought about this as well but then decided to apply the terminology used for Kubernetes TLS secrets.
It could be changed if you feel strongly about this.